src/client/Client.cc

```diff
@@ -4073,8 +4065,8 @@ void Client::send_flush_snap(Inode *in, MetaSession *session,
   auto m = make_message<MClientCaps>(CEPH_CAP_OP_FLUSHSNAP,
 				     in->ino, in->snaprealm->ino, 0,
 				     in->auth_cap->mseq, cap_epoch_barrier);
-  m->caller_uid = capsnap.cap_dirtier_uid;
-  m->caller_gid = capsnap.cap_dirtier_gid;
+  m->caller_uid = -1;
+  m->caller_gid = -1;
```

Comment:
nit: could you paste the commit message as a comment here as well; it'll help explain things quicker